core/templates/dev/head/domain/statistics/LearnerActionObjectFactorySpec.js

```diff
@@ -0,0 +1,51 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the LearnerActionObjectFactory.
+ */
+
+describe('Learner Action Object Factory', function() {
+  var LearnerActionObjectFactory;
```

Comment:
Same, please use [this](https://github.com/CareMessagePlatform/jasmine-styleguide#this-is-how-we-do-it).

Reply:
Done.